### PR TITLE
Restapi endpoint support

### PR DESCRIPTION
**Pull request by nileshgulia1**
The endpoint was originally extracted from https://github.com/eea/eea.api.taxonomy. Continuing the work done by @cekk. 
The idea is to utilise this endpoint in volto-addon controlpanel https://github.com/eea/volto-taxonomy/tree/develop

### CODE REVIEW

**Review by MrTango — APPROVED** [review]
with the small cleanup comments, it looks good to me

File: .installed.txt
[the remark this answers is not in the document]
Comment (petschki) [reply .installed.txt:1]: no

**Review by petschki — CHANGES_REQUESTED** [review]
would be nice if the README would be updated too.

File: .DS_Store
[the remark this answers is not in the document]
Comment (nileshgulia1) [reply .DS_Store:1]: Removed in [b6da62a](https://github.com/collective/collective.taxonomy/pull/126/commits/b6da62a4a77c6dfe46ff56b40c85ef484a38225e)

**Comment by MrTango** [comment]
@nileshgulia1 thx

**Pull request merged**